config/set.c

```diff
@@ -261,7 +261,7 @@ bool cs_register_type(struct ConfigSet *cs, unsigned int type, const struct Conf
 bool cs_register_variables(const struct ConfigSet *cs, struct ConfigDef vars[], int flags)
 {
   if (!cs || !vars)
-    return CSR_ERR_CODE; /* LCOV_EXCL_LINE */
+    return false;
```

Comment:
We should document this return value.

Reply:
`cs` and `vars` are mandatory and if either is missing the result is a failure.
The retval is type `bool` and I declare:
```
// @retval bool True, if all variables were registered successfully
```

I can change it, if you really think it adds something:
```
// @retval true  All variables were registered successfully
// @retval false Not all variables were registered successfully
```